examples/analysis/point_extraction/interpolation.py

```diff
@@ -0,0 +1,55 @@
+"""
+Regular-grid interpolation
+==========================
+
+This example demonstrates raster interpolation to point values using :func:`~geoutils.Raster.interp_points`.
+"""
+# sphinx_gallery_thumbnail_number = 2
```

Comment:
This line gets rendered, it probably shouldn't be?

Reply:
Solved in [9d487d0](https://github.com/GlacioHack/geoutils/pull/351/commits/9d487d0a85fe2acffbeb4b75d2e3765d23090f66)